core/templates/dev/head/pages/topic_viewer/StoriesListDirective.js

```diff
@@ -0,0 +1,132 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the stories list.
+ */
+oppia.directive('storiesList', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {
+        getTopicId: '&topicId',
+        getCanonicalStories: '&canonicalStoryList',
```

Comment:
We're calling this attr storyList, but we're calling the directive storiesList. Let's be consistent and pick one phrasing, and use that throughout from backend to frontend.

Reply:
Changed.